tests/sepp/Options.h

```diff
@@ -76,15 +77,17 @@ auto inspect(Inspector& f, Setup& o) {
                             f.field("prefill", o.prefill).fallback(f.keep()));
 }
 
-using WorkloadVariants = std::variant<workloads::GetByPrimaryKey::Options,
-                                      workloads::InsertDocuments::Options,
-                                      workloads::IterateDocuments::Options>;
+using WorkloadVariants = std::variant<
+    workloads::WriteWriteConflict::Options, workloads::GetByPrimaryKey::Options,
+    workloads::InsertDocuments::Options, workloads::IterateDocuments::Options>;
 namespace workloads {
 // this inspect function must be in namespace workloads for ADL to pick it up
 template<class Inspector>
 inline auto inspect(Inspector& f, WorkloadVariants& o) {
   namespace insp = arangodb::inspection;
   return f.variant(o).unqualified().alternatives(
+      insp::type<workloads::WriteWriteConflict::Options>(
+          "write-write-conflict"),
       insp::type<workloads::GetByPrimaryKey::Options>("getByPrimaryKey"),
```

Comment:
Casing is incosistent here: `write-write-conflict` vs `getByPrimaryKey`. We should go with only convention here.